components/SearchBar/styles.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+export const SearchBarContainer = styled.form`
+  display: flex;
+  width: 296px;
+  height: 17px;
+  padding: 8px 16px;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 16px;
+  border: 1px solid #A6A6A6;
+  background: #FFF;
+  position: relative;
+  left: 45px;
+`;
+
+export const SearchInput = styled.input`
+  flex: 1;
+  border: none;
+  background: transparent;
+  outline: none;
+`;
+
+export const IconWrapper = styled.button`
+  background: none;
+  border: none;
+  position: absolute;
+  right: 8px;
+  top: 50%;
+  transform: translateY(-50%);
```

Comment:
Also, so seem to be mixing display `flex` and position absolute. 

If i look at the code above it seems like you want to make the form a flex container. And then the input and the button as flex children. If we do that then we can remove the position absolute and the positioning related to that. 

```suggestion

```